src/components/List/List.tsx

```diff
@@ -62,7 +64,7 @@ export const List = ({
               {rows.map((row: any, idx: number) => (
                 <TableRow
                   key={idx}
-                  sx={{ '&:last-child td, &:last-child th': { border: 0 } }}
+                  sx={{ '&:last-child td, &:last-child th': { border: 0 } , height : height }}
                 >
```

Comment:
This height is an optional parameter right.
So we need to handle the case when it is `undefined` also and in such case the `sx` should basically not have a height attribute. 

Maybe use the javascript spread operation `...` to conditonally add height when its defined

Reply:
Super !! Thanks for making the changes